src/ethz_snow/constants.py

```diff
@@ -200,6 +231,107 @@ def calculateDerived(fpath: Optional[str] = None) -> dict:
         "beta_solution",
     ]
 
+    if config["configuration"] == "VISF":
+        if config["temperature"] == "homogeneous":
+            raise NotImplementedError(
+                (
+                    f'For simulating "{config["configuration"]}" '
+                    + "a spatial model is required."
+                )
+            )
+
+    # set up additional parameters for VISF
+    p_vac = float(config["VISF"]["p_vac"])
+    kappa = float(config["VISF"]["kappa"])
+    Dh_evaporation = float(config["VISF"]["Dh_evaporation"])
+    m_water = float(config["VISF"]["m_water"])
+    t_vac_start = float(config["VISF"]["t_vac_start"])
+    t_vac_duration = float(config["VISF"]["t_vac_duration"])
+
+    constVars.extend(
+        [
+            "p_vac",
+            "kappa",
+            "Dh_evaporation",
+            "m_water",
+            "t_vac_start",
+            "t_vac_duration",
+        ]
+    )
+
+    if config["configuration"] == "jacket":
+        if config["temperature"] != "spatial_2D":
+            raise NotImplementedError(
+                (
+                    f'For simulating "{config["configuration"]}" '
+                    + "a 2D model is required."
+                )
+            )
+
+    # set up additional parameters for VISF
+    air_gap = float(config["jacket"]["air_gap"])
+    lambda_air = float(config["jacket"]["lambda_air"])
+
+    constVars.extend(["lambda_air", "air_gap"])
+
+    if config["temperature"] != "homogeneous":
+        if config["vial"]["geometry"]["shape"] != "cylinder":
+            raise NotImplementedError(
+                (
+                    f'For simulating "{config["temperature"]}" '
+                    + "a cylindrical geometry is required."
+                )
+            )
+
+        # this is not a lumped capacitance model
+        # effective heat conductivity (for non-homog. vial temps)
+        lambda_s = float(config["solution"]["lambda_s"])
+        lambda_w = float(config["water"]["lambda_w"])
+        lambda_i = float(config["water"]["lambda_i"])
+
+        # latent heat of fusion
+        Dh = float(config["water"]["Dh"])
+
+        # needed for solving nucleation
+        k_f = float(config["solution"]["k_f"])
+        M_s = float(config["solution"]["M_s"])
+
+        # heat conductivity
```

Comment:
confusing/contradicting comments above and below line

Reply:
Corrected.